darwin/utils.py

```diff
@@ -207,6 +245,8 @@ def parse_darwin_annotation(annotation: dict):
         main_annotation = dt.make_line(name, annotation["line"]["path"])
     elif "keypoint" in annotation:
         main_annotation = dt.make_keypoint(name, annotation["keypoint"]["x"], annotation["keypoint"]["y"])
+    elif "skeleton" in annotation:
+        main_annotation = dt.make_skeleton(name, annotation["skeleton"]["nodes"])
```

Comment:
This is totally fine by me. Remember we don't have the annotation class metadata, where the definitions of nodes and edges are stored. For that, we had a manifest PR, which I don't think we ever got to merge @simedw